viewing_party/party.py

```diff
@@ -1,23 +1,179 @@
 # ------------- WAVE 1 --------------------
 
+from re import I

```

Comment:
VS Code will try to auto-import libraries a bit too speculatively. I didn't see any use of this function in the code below, so I'm guessing it was an accident. It's always a good idea to look carefully at one's PR before submitting it to a reviewer, so it's more likely one would catch things like this. I believe I turned off the auto-import feature of VS Code in the settings to avoid things like this.